editor-extensions/vscode/client/src/test/helper.ts

```diff
@@ -0,0 +1,47 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import * as vscode from 'vscode';
+import * as path from 'path';
+
+export let doc: vscode.TextDocument;
+export let editor: vscode.TextEditor;
+export let documentEol: string;
+export let platformEol: string;
+
+/**
+ * Activates the vscode.lsp-sample extension
+ */
+export async function activate(docUri: vscode.Uri) {
+	// The extensionId is `publisher.name` from package.json
+	const ext = vscode.extensions.getExtension('vscode-samples.lsp-sample')!;
```

Comment:
Should we update this now, or are we just leaving this in until we've got tests?

Reply:
Updated to reflect this package, but I'll leave this here.  There are no tests at the moment as this is such a simple wrapper, but they will be coming for the autocomplete and other work, and will be much more useful once this is re-implemented in Reason on the server side.